src/simplify_algebra.cpp

```diff
@@ -359,6 +402,8 @@ void simplify_algebra::apply(program& p) const
                             find_mul_conv{},
                             find_mul_add{},
                             find_concat_unary{},
+                            find_div_const{},
+                            find_sub_const{},
```

Comment:
Dont put this inbetween the concat finders. Ideally, just put it before `find_concat_unary`.

Reply:
fixed